models/recordsModel.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from "mongoose";
+const recordSchema = new mongoose.Schema(
+  {
+    questionId: { type: String, required: true, unique: false },
+    answers: [{ type: Number, requred: true }]
+  },
+  { timestamps: true }
+);
+
+const RecordModel = mongoose.model("Record", recordSchema);
+
+const save = async model => new RecordModel(model).save();
+const getAllRecords = async () => RecordModel.find();
+const getRecordById = async recordId => RecordModel.findOne(recordId);
+
+const updateById = async (recordId, update) =>
+  RecordModel.findOneAndUpdate(recordId, update, { new: true });
+
+const deleteRecordById = async recordId =>
+  RecordModel.findOneAndDelete(recordId);
+
+// What acctual data we need to handle throuw
+const getRecordsFromIdArray = async recordId =>
+  RefordModel.find({
+    recordID: { $in: [mongoose.Types.ObjectId(recordId.map())] }
```

Comment:
This wont work for few reasons.

- You need to specify what field you want to make query agains. I think it should be ID
- You are using map function but I think you do not know how it work. I would suggest creating array before and then pass it into function. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map